en/appendices/4-0-upgrade-guide.rst

```diff
@@ -52,7 +52,7 @@ plugin:
     bin/cake upgrade file_rename templates <path/to/app>
 
 Once you've renamed your template and locale files, make sure you update
-``App.paths.locales`` and ``App.paths.templates`` paths to be correct.
+``App.paths.locales`` and ``App.paths.templates`` (in ``/config/app.php``) paths to be correct.
```

Comment:
```suggestion
``App.paths.locales`` and ``App.paths.templates`` paths (in **/config/app.php**) to be correct.
```

Reply:
Updated to be after `paths`. I cannot commit this fix to merge though.

Reply:
why are we using bold here? I think the code fencing is cleaner.

Reply:
Paths are bold generally in the docs. inline code is for classes, variables and methods.